client/src/views/components/CreateProductSlide/CategoryList/CategoryList.js

```diff
@@ -27,19 +31,20 @@ export default function CategoryList({ $target, onClickHandler, initialState })
   };
 
   this.render = () => {
-    this.convertToHTML();
+    const html = this.convertToHTML();
+    this.$categoryList.innerHTML = html;
   };
 
   this.convertToHTML = () => {
     const html = this.category_list
       .map((category) => {
```

Comment:
코드를 보니 변수 할당안하고 즉시 return 해도 될것 같네요!!
가독성을 위해서라면 좋습니다 :)

```suggestion
  this.convertToHTML = () => this.category_list.map((category) => { ... 
```

Reply:
함수명으로도 이해가 가능해서 불필요한 변수선언이었던 것 같네요!
변수를 빼겠습니다!